js/controllers/settings.js

```diff
@@ -60,6 +60,28 @@ SettingsController.prototype.show_ = function(key, value) {
   }
 };
 
+/**
+ * Activates a switch Material Component element in the UI.
+ * @param {string} key The unique section of the id of the switch element
+ *     (after the 'setting-' prefix).
```

Comment:
Add `@private` to both this and the other method.

Reply:
Done.